ci-operator/config/openshift/metallb/openshift-metallb-main.yaml

```diff
@@ -49,25 +49,31 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
-- as: metallb-l2-e2e-metal-ipi
+- as: metallb-e2e-metal-ipi
   steps:
     cluster_profile: packet
     env:
       DEVSCRIPTS_CONFIG: |
         IP_STACK=v4
         NETWORK_TYPE=OpenShiftSDN
-        ENABLE_METALLB_MODE=l2
+        ENABLE_METALLB=true
         METALLB_REPO=https://github.com/openshift/metallb.git
+        METALLB_OPERATOR_REPO=https://github.com/openshift/metallb-operator.git
+        METALLB_OPERATOR_COMMIT=22afa1a
```

Comment:
Why hard-coding to a commit SHA?
Can't METALLB_OPERATOR_COMMIT be set to "main" so that it keeps up with latest main branch?

Reply:
if we do a change in the operator that's not reflected in metallb we might break the CI

Reply:
I don't follow. Could you please share an example?

Reply:
so this operator commit id is for deploying metallb through the operator is d/s metallb CI right? passed to the devscripts here
https://github.com/openshift-metal3/dev-scripts/blob/918da5cba37ec38b47e67dbe08d53297e8dcb7ea/metallb/deploy_operator.sh#L7
if we do a big change in the operator it will immediately affect the deployment of the operator for metallb CI, so it might break all our PRs in metallb repo

Reply:
So your example is of a potential disruptive MetalLB Operator PR that breaks MetalLB deployments, correct?
If such a big change in the operator breaks MetalLB CI, it should be caught by CI on the MetalLB Operator PR itself before it is merged in the first place and impacts the MetalLB CI. Am I missing something?

Reply:
@fedepaol what do you think?

Reply:
The way it works is, a new PR on the operator (or on the frr image, or on metallb) builds the images, and uses whatever it is in terms of manifests in the operator to deploy and test them. This is not optimal, but will cover the 90% of our cases.
@sabinaaledort 's concern is legit, and we won't able to block a bad PR from the operator. On the other hand, having everything broken will force us to fix it :-).

So I'd choose pinning to a branch.

Reply:
so branch it is :)